src/machine/machine_atsamd51.go

```diff
@@ -141,6 +141,13 @@ func (p Pin) Get() bool {
 	return (sam.PORT.GROUP[group].IN.Get()>>pin_in_group)&1 > 0
 }
 
+// Invert the current value of a GPIO pin.
```

Comment:
The Go convention is to start the comment with the function name itself. I would suggest:

```go
// Toggle switches an output pin from low to high or from high to low.
```

Reply:
Fixed.  Good catch.